pkg/storage/allocator_test.go

```diff
@@ -685,6 +689,124 @@ func TestAllocatorRebalance(t *testing.T) {
 	}
 }
 
+// TestAllocatorRebalanceTarget could help us to verify whether we'll rebalance to a target that
+// we'll immediately remove.
+func TestAllocatorRebalanceTarget(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	manual := hlc.NewManualClock(123)
+	clock := hlc.NewClock(manual.UnixNano, time.Nanosecond)
+	// We're going to manually mark stores dead in this test.
+	stopper, g, _, a, _ := createTestAllocator( /* deterministic */ false)
+	defer stopper.Stop(context.Background())
+	stores := []*roachpb.StoreDescriptor{
```

Comment:
Thanks for explaining in the code review, but a comment in the code explaining this configuration of stores would be very helpful to future maintainers.

Reply:
Sorry for that. I have already added a comment about the configuration of stores.